File: Proyecto/server/BusinessLayer/Usuario.py
from DataLayer.UsuarioDB import *
from EntityLayer.UsuarioEntity import *


class Usuario:
    def Save(Ent: UsuarioSaveModel):
        try:
            return UsuarioDB.Save(Ent)
        except Exception as e:
            print(e)
    
    def GetItems():
        try:
            return UsuarioDB.GetItems()
        except Exception as e:
            print(e)
    
    def GetItem(Id: int):
        try:
            return UsuarioDB.GetItem(Id)
        except Exception as e:
            print(e)
    
    def Delete(Id: int):
        try:
            return UsuarioDB.Delete(Id)
        except Exception as e:
            print(e)
    

File: Facturacion/FactCore/FactCore.BusinessLayer/Usuario.cs
using FactCore.DataLayer;
using FactCore.EntityLayer;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FactCore.BusinessLayer
{
    public class Usuario
    {


        public static List<UsuarioEntity> OObtenerAcceso(String Usuario, String Contrasena)
        {
            UsuarioDB DB = new UsuarioDB();
            return DB.ObtenerAcceso(Usuario, Contrasena);
        }
    }
}
